spec/requests/errors_controller_spec.rb

```diff
@@ -75,4 +75,28 @@
   describe 'GET #serice_unavailable' do
     it_behaves_like 'a csv or json error response', 503, 'Service is unavailable'
   end
+
+  describe 'GET #maintenance' do
```

Comment:
`def maintenance` didn't have a spec.

Reply:
We've already got a separate feature spec for maintenance mode so this isn't required, see

`spec/features/maintenance_mode_spec.rb`